Source/Tools/PR.viancfg

```diff
@@ -0,0 +1,1062 @@
+		<Selected>FALSE</Selected>
+		<Controls>
+		</Controls>
+	</Test>
+	<Test>
+		<Name>"Control Terminal Label Visible"</Name>
+		<Ranking>0</Ranking>
+		<MaxFailures>5</MaxFailures>
+		<BasePath>"LabVIEW"</BasePath>
+		<RelativePath>"project\\_VI Analyzer\\_tests\\Block Diagram\\Style\\Control Terminal Label Visible.llb"</RelativePath>
+		<Selected>FALSE</Selected>
+		<Controls>
+		</Controls>
+	</Test>
+	<Test>
+		<Name>"Control Terminal Wiring"</Name>
+		<Ranking>0</Ranking>
+		<MaxFailures>5</MaxFailures>
+		<BasePath>"LabVIEW"</BasePath>
+		<RelativePath>"project\\_VI Analyzer\\_tests\\Block Diagram\\Style\\Control Terminal Wiring.llb"</RelativePath>
+		<Selected>FALSE</Selected>
+		<Controls>
+		</Controls>
+	</Test>
+	<Test>
+		<Name>"Diagram Disable Structures"</Name>
+		<Ranking>0</Ranking>
+		<MaxFailures>5</MaxFailures>
+		<BasePath>"LabVIEW"</BasePath>
+		<RelativePath>"project\\_VI Analyzer\\_tests\\Block Diagram\\Style\\Diagram Disable Structures.llb"</RelativePath>
+		<Selected>FALSE</Selected>
+		<Controls>
+		</Controls>
+	</Test>
+	<Test>
+		<Name>"Mixed Terminal Styles"</Name>
+		<Ranking>0</Ranking>
+		<MaxFailures>5</MaxFailures>
+		<BasePath>"LabVIEW"</BasePath>
+		<RelativePath>"project\\_VI Analyzer\\_tests\\Block Diagram\\Style\\Mixed Terminal Styles.llb"</RelativePath>
+		<Selected>FALSE</Selected>
+		<Controls>
+		</Controls>
+	</Test>
+	<Test>
+		<Name>"Poor Names of Enum Items"</Name>
+		<Ranking>1</Ranking>
+		<MaxFailures>5</MaxFailures>
+		<BasePath>"LabVIEW"</BasePath>
+		<RelativePath>"project\\_VI Analyzer\\_tests\\Block Diagram\\Style\\Poor Names of Enum Items.llb"</RelativePath>
+		<Selected>FALSE</Selected>
+		<Controls>
+		</Controls>
+	</Test>
+	<Test>
+		<Name>"Sequence Structure Usage"</Name>
+		<Ranking>1</Ranking>
+		<MaxFailures>5</MaxFailures>
+		<BasePath>"LabVIEW"</BasePath>
+		<RelativePath>"project\\_VI Analyzer\\_tests\\Block Diagram\\Style\\Sequence Structure Usage.llb"</RelativePath>
+		<Selected>FALSE</Selected>
+		<Controls>
+		</Controls>
+	</Test>
+	<Test>
+		<Name>"String Constant Style"</Name>
+		<Ranking>0</Ranking>
+		<MaxFailures>5</MaxFailures>
+		<BasePath>"LabVIEW"</BasePath>
+		<RelativePath>"project\\_VI Analyzer\\_tests\\Block Diagram\\Style\\String Constant Style.llb"</RelativePath>
+		<Selected>FALSE</Selected>
+		<Controls>
+		</Controls>
+	</Test>
+	<Test>
+		<Name>"Tunnel Position"</Name>
+		<Ranking>1</Ranking>
+		<MaxFailures>5</MaxFailures>
+		<BasePath>"LabVIEW"</BasePath>
+		<RelativePath>"project\\_VI Analyzer\\_tests\\Block Diagram\\Style\\Tunnel Position.llb"</RelativePath>
+		<Selected>FALSE</Selected>
+		<Controls>
+		</Controls>
+	</Test>
+	<Test>
+		<Name>"Unused Code"</Name>
+		<Ranking>2</Ranking>
+		<MaxFailures>5</MaxFailures>
+		<BasePath>"LabVIEW"</BasePath>
+		<RelativePath>"project\\_VI Analyzer\\_tests\\Block Diagram\\Style\\Unused Code.llb"</RelativePath>
+		<Selected>FALSE</Selected>
```

Comment:
This VI Analyzer configuration is set to run most tests with a default `<Ranking>2</Ranking>` (high severity) and not run tests with `<Ranking>1</Ranking>` (medium severity) or `<Ranking>0</Ranking>` (low severity).  The only exceptions are three disabled high-severity tests:
- `Unused Code` disabled due to some of grpc-labview's generated code having unused unbundle nodes
- `Broken VI` disabled due to a handful of flagged cases.
  - We have a G test to cover broken VIs too.  However, perhaps we should strive to enable the broken VI coverage via VI Analyzer, then perhaps the G test could be removed..?
- `Reentrant VI Issues` disabled due to a number of flagged cases, most of which involve a non-reentrant subVI being called from a reentrant caller.

Reply:
I'm surprised that VI Analyzer caught broken VIs but our test didn't. In any case, I agree we should rely on VI Analyzer and not our test. However, we should do that as part of another PR.

Reply:
I think the problem actually may have been that I didn't have all the drivers on my local develoment setup.  Maybe the check would work when run on spawningpool.

Reply:
I'll try it out in a follow-up PR :)

Reply:
Sounds good. We do our best to make sure nothing is broken, even if a driver isn't installed. However, there is one private VISA class VI that will be broken if you don't have VISA 24.0 installed. Since this breakage never propagates to customers and since there isn't an ideal fix, we left it as is. But it would definitely trip up VI analyzer.